packages/payload/src/admin/components/views/Login/index.tsx

```diff
@@ -75,22 +75,32 @@ const Login: React.FC = () => {
               action={`${serverURL}${api}/${userSlug}/login`}
               className={`${baseClass}__form`}
               disableSuccessStatus
-              initialData={{
-                email: autoLogin && autoLogin.prefillOnly ? autoLogin.email : undefined,
-                password: autoLogin && autoLogin.prefillOnly ? autoLogin.password : undefined,
-              }}
+              initialData={
```

Comment:
Hmm it might be a little more readable to optionally chain `autoLogin?.prefillOnly` and use a ternary to conditionally spread the credentials objcet:

```suggestion
              initialData={
                {...autoLogin?.prefillOnly ? {
                  email: autoLogin.email,
                  password: autoLogin.password
                } : {}}
               }
```